### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by Medico381** [community/community#130831] [New to GitHub]
I am new user I want to know how can I use this ? And I am here for practice so what should I do?

**Comment by felipeserrano05** [comment]
what do you mean by that? You can use the GitHub as a storage manager for storing your repositories.

**Comment by supolaris** [comment]
First learn the basic usage from any youtube tutorial. and github is not for practice code but to mange and store all of code and you can collaborate your team from github

**Comment by ghostinhershell** [comment]
Welcome to GitHub and the GitHub Community @Medico381 🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!
